01-javascript/06-async-await/Manasi_async_await.js

```diff
@@ -0,0 +1,36 @@
+const { readFile } = require('fs');
+
+//variables to check whether there is an error while reading the file
+var lolfound = false;
+var sasafound = false;
+
+//function returns a promise
+function fileReaderPromise(path) {
+    return new Promise((res,rej)=>{
```

Comment:
can you please format you code

Reply:
I have formatted the code. Please let me know if any further changes are required.